app/src/main/java/com/example/moviedb/ui/base/BaseActivity.kt

```diff
@@ -1,7 +1,26 @@
 package com.example.moviedb.ui.base
 
+import android.os.Bundle
 import androidx.appcompat.app.AppCompatActivity
+import androidx.databinding.DataBindingUtil
+import androidx.databinding.ViewDataBinding
+import androidx.lifecycle.LifecycleOwner
+import com.example.moviedb.data.model.Movie
 
-abstract class BaseActivity: AppCompatActivity() {
+abstract class BaseActivity<ViewBinding : ViewDataBinding, ViewModel : BaseViewModel<Movie>> : AppCompatActivity(),
+    LifecycleOwner {
```

Comment:
Sao phải kế thừa cái này nhỉ?

Reply:
cái này lúc nãy t viết BaseActivity đoạn 
        viewBinding.lifecycleOwner = this
nó k nhận this theo lifecycler nên nó suggest là impl cái LifecyclerOwner đó :)